### PR TITLE
Chart Column Interval Config

### PR DESCRIPTION
**Pull request by llvasconcellos**
This is the server side implementation for the pivotal story:
***Allow profile for a chart to specify one column per encounter (or per minute)***

* Added the profile format column to the [chart_divider] field description.

This is the profile I'm using for testing it:

[nutrition-atfc-101-v3.txt](https://github.com/projectbuendia/buendia/files/150180/nutrition-atfc-101-v3.txt)

### CODE REVIEW

**Comment by dancunningham** [comment]
Great! So this will stop notes appearing over and over again?

What is the [chart_divider] thing used for anyway? Seems like a pretty nasty hack that you're working around here. Would there be a better way?

**Comment by llvasconcellos** [comment]
@dancunningham: 
"Great! So this will stop notes appearing over and over again?" What do you mean? 
[chart_divider] is used for multiple charts. See the chart order form on OpenMRS. I did it this way to deliver it faster due to time constrictions. We could do a more polished way adding more complexity.
I don't think it's nasty though - It's pretty straightforward.

**Comment by capnfabs** [comment]
@dancunningham @llvasconcellos the "Notes" field was appearing in every chart for printed charts. I fixed that in the `feature/llvasconcellos-printable-charts` branch.

**Comment by dancunningham** [comment]
What's the impact this particular PR is intended to have?

**Comment by llvasconcellos** [comment]
@dancunningham This is part of the story I'm working on. 
***Allow profile for a chart to specify one column per encounter (or per minute)***

File: tools/profile_apply [hunk 1]
```diff
@@ -276,7 +276,12 @@ def apply(tabs):
                 section, section_rows = row['section'].strip(), []
             if row['title'].strip():
```
Comment (dancunningham) [inline tools/profile_apply]: Can you add a comment in the code to explain what is going on here please?

It's not clear just from the code that when a new title is found, it creates a chart divider with a dummy concept ID and the format will contain the column type and interval details.

Is it necessary to have a dummy concept ID for a section? I'm a bit uneasy about it.

**Comment by dancunningham** [comment]
Now I understand what this is for :-)

**Pull request merged**